git-gutter.el

```diff
@@ -608,7 +608,7 @@ Argument TEST is the case before BODY execution."
             (make-local-variable 'git-gutter:diffinfos)
             ;;(setq-local git-gutter:start-revision nil)
             (add-hook 'kill-buffer-hook 'git-gutter:kill-buffer-hook nil t)
-            (add-hook 'pre-command-hook 'git-gutter:pre-command-hook)
+            (add-hook 'pre-command-hook 'git-gutter:pre-command-hook t)
```

Comment:
@hikmet517 this only sets the `DEPTH` in `add-hook`, it does not set `LOCAL`. 

The signature for `add-hook` is `(add-hook HOOK FUNCTION &optional DEPTH LOCAL)`